src/main/java/ch/njol/skript/expressions/ExprDrops.java

```diff
@@ -94,67 +94,105 @@ public Class<?>[] acceptChange(ChangeMode mode) {
 			case REMOVE_ALL:
 			case SET:
 				return CollectionUtils.array(ItemType[].class, Inventory[].class, Experience[].class);
-			case DELETE:
+			case DELETE: // handled by EffClearDrops
```

Comment:
maybe we should change this in the future. is there a good reason to not support this?